src/components/Inputs/CheckBox.js

```diff
@@ -10,15 +10,8 @@ export default class CheckBox extends Component {
     this.onInputChange = this.onInputChange.bind(this);
 
     }
-    onInputChange(event){
-        if(this.state.toggle){
-            this.setState({toggle:false})
-            this.props.updateCB(this.props.elementName, false);
-        }else{
-            this.setState({toggle:true})
-            this.props.updateCB(this.props.elementName, true);
-
-        }
+    onInputChange(_event){
+            this.setState({toggle: !this.state.toggle})
```

Comment:
this no longer updates the state, makes checkboxes useless

Reply:
Whoops, I missed that. Added back as of [721b6b2](https://github.com/mcode/request-generator/pull/53/commits/721b6b2e25e78cb8453d74ce25336a9c83d243dc).